SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -0,0 +1,1241 @@
+
+%global security_hardening none
+%global sha512hmac bash %{_sourcedir}/sha512hmac-openssl.sh
+%define uname_r %{version}-hvl1.m2
+%ifarch x86_64
+%define arch x86_64
+%define archdir x86
+%define config_source %{SOURCE1}
+%endif
+
+Summary:        Mariner kernel that has dom0 support and works with MSHV
```

Comment:
Please use "MSHV Host support" instead of "dom0 support".

But it's OK if you decide to make this change later.